.travis.yml

```diff
@@ -0,0 +1,51 @@
+language: node_js
+cache: npm
```

Comment:
yarn?

Reply:
also, to speed up build, specify node_modules directory like in Lodestar so it skips installing if everything was installed in some previous build